src/test/java/mops/Material1ApplicationTests.java

```diff
@@ -1,12 +1,20 @@
 package mops;
 
+import mops.businesslogic.GroupService;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 
+@SuppressWarnings("PMD")
 @SpringBootTest
 class Material1ApplicationTests {
+    /**
+     * Necessary mock until GroupService is implemented.
+     */
+    @MockBean
+    private GroupService groupService;
```

Comment:
Why do we need the GroupService here?

Reply:
Spring will look for a bean and will not find on. That's what I tried to explain in the comment.